docs/Administrator Guide/arbiter-volumes-and-quorum.md

```diff
@@ -194,13 +195,6 @@ Say B1, B2 and B3 are the bricks:
 that B2 and B3's pending xattrs blame each other and therefore the write is not
 allowed and is failed with an EIO.
 
-There is a corner case even with replica 3 volumes where the file can
```

Comment:
Also remove the mention of this corner case @213

Reply:
@karthik-us  Done.